app/main.py

```diff
@@ -1,13 +1,21 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    return {
+        ["name", "source", "line", "column", "message"][i]:
+        ("flake8" if key == "filename" else error[key])
+        for i, key in enumerate(error) if key != "physical_line"
+    }
```

Comment:
you should hardcode the dictionary the same way as you did in def format_single_linter_file